import "rec/command/Command.proto";

package rec.command;

message CommandMapEntry {
  optional Command.Type command = 1;
  repeated bytes key = 2;
};

message CommandMapProto {
  repeated CommandMapEntry entry = 1;
};

message KeyStrokeCommandMapProto {
  optional CommandMapProto map = 1;
};

message MidiCommandMapProto {
  optional CommandMapProto map = 1;
};